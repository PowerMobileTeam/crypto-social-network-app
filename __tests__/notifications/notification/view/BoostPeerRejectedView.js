import 'react-native';
import React from 'react';
import { Text, TouchableOpacity } from "react-native";
import { shallow } from 'enzyme';
import BoostPeerRejectedView from '../../../../src/notifications/notification/view/BoostPeerRejectedView';
import styles from '../../../../src/notifications/notification/style';

// fake data generation
import boostNotificationFactory from '../../../../__mocks__/fake/notifications/BoostNotificationFactory';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {

  const entity = boostNotificationFactory('boost_peer_rejected');

  const notification = renderer.create(
    <BoostPeerRejectedView styles={styles} entity={entity}/>
  ).toJSON();

  expect(notification).toMatchSnapshot();
});