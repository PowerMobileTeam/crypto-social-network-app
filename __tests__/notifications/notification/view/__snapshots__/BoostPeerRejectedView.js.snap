// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "flexWrap": "wrap",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    onPress={[Function]}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      onPress={[Function]}
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      @
      someUser
    </Text>
     declined your bid of 
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      100
       
      points
    </Text>
     
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        for 
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#444",
            "fontWeight": "bold",
          }
        }
      >
        some title
      </Text>
    </Text>
    You have not been charged.
  </Text>
</View>
`;
